src/Propel/Runtime/Map/DatabaseMap.php

```diff
@@ -143,11 +143,11 @@ public function addTableFromMapClass(string $tableMapClass): TableMap
      * {@link DatabaseMap::getTableByPhpName()}, or
      * {@link DatabaseMap::getTables()}
      *
-     * @param class-string $tableMapClass The name of the table map to add
+     * @param class-string<\Propel\Runtime\Map\TableMap>|string $tableMapClass The name of the table map to add
```

Comment:
|string is from sniffer?
We can adjust the sniffer to allow this in a future fix/PR

Reply:
Yes, I think that would be nice

Reply:
See https://github.com/spryker/code-sniffer/commit/93cdaa1257fb7c7d7a425fba943316f77f5d9a4d

Reply:
Lets keep it for now until we tag a new sniffer release, then we can clean up

Reply:
Nice!